extras/oh-my-posh/README.md

```
@@ -0,0 +1,74 @@
+# Aquaposh.omp 🐚
+##### Aquavium color scheme on Oh My Posh
```

markdownlint-cli2 (0.22.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h5

(MD001, heading-increment)